src/api/goods.ts

```diff
@@ -5,15 +5,28 @@ const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/good
 
 export function getAll(): Promise<Good[]> {
   return fetch(API_URL)
-    .then(response => response.json());
+    .then(response => {
+      if (!response.ok) {
+        throw new Error(`${response.status} - ${response.statusText}`);
+      }
+
+      if (response.headers.get('content-type')?.includes('aplication/json')) {
```

Comment:
There is a typo in the content type 'aplication/json'. It should be 'application/json' 🚩

Reply:
Thanks